app/views/posts/_form.html.erb

```diff
@@ -6,6 +6,7 @@
   <title>Editor</title>
   <style type="text/css" media="screen">
     html {
+       <!-- able webpage to scroll -->
```

Comment:
The code looks good and should work. Maybe what you mean in the comment is "enable webpage to scroll"?